res/values/colors.xml

```diff
@@ -1,3 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+    <color name="oem_unlocked_warning_color">@android:color/holo_orange_dark</color>
```

Comment:
Holo styling has been long deprecated and will likely be removed at some point.

Reply:
removed reference